scene/2d/path_2d.cpp

```diff
@@ -140,11 +148,15 @@ void Path2D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_curve", "curve"), &Path2D::set_curve);
 	ClassDB::bind_method(D_METHOD("get_curve"), &Path2D::get_curve);
 	ClassDB::bind_method(D_METHOD("_curve_changed"), &Path2D::_curve_changed);
+	ClassDB::bind_method(D_METHOD("set_frame_color", "color"), &Path2D::set_frame_color);
+	ClassDB::bind_method(D_METHOD("get_frame_color"), &Path2D::get_frame_color);
 
+	ADD_PROPERTY(PropertyInfo(Variant::COLOR, "pathColor"), "set_frame_color", "get_frame_color");
```

Comment:
the property should be `path_color` and as you mentioned the setter and getter name should be set/get_path_color